.travis.yml

```diff
@@ -13,8 +13,8 @@ env:
   global:
   - CXX=g++-4.8
   - DEBUG='* -superagent -express:* -mocha:* -babel -nock* -eslint*'
-  - secure: CFHtd3es0aG0YgsA5OBVgdLZlIrdHdEhayvyJSCWo0AooyVe1zxbBtAPLWysm3UeaK5iqDAPZXrH/M+GXiLFE+yiSkJk+O7q6Fx7rM3oO3J1AZLbseuCbdjJGe/2NXDDzadfeV/4Z9aDYf6f17jxYNlq33XZvNHLBr2BbXw5DzZrcIp4qa8wZym/3Gmp9un9rR3+Pij5iNItaTRpjYQYug12GQsueMaB9dgqCpKVAWzGfOE5zDH1ilyyq9O1xdaRDEeFaPmMzh47a1nCBJ/OcCra21PRDa3BKjnemrGM3TQhTdX2JN5QZTKcDvg9XiapafsIMisbsG8U2GHB61y0T64HZzHLa3lhq0LVfj8aPH7x8cruDMIibB4mKeScgN1/VSxJqwdisPJMYnfX8z4N/7WI53a+nfaVjd7q7kbBQRzmI8zQVdMFzJ2pcl2aISGyl6ZD8EFjQLJQzPJixMpbAPz++0PeBJ76yBYJvD1NshJTLFwjPken+5bQT0ltKS2KuYgkXxS0dVX4nXsJFuUXqBtqEm8vuqh0+tQqjMY6GDsGk6DC6KCHaKfEPhYSWapVhO4F15PArydm1APQ6EhJLB7I0tm1LV0FAcj884J8vWPmwGCakqJ09SmxXTtIyepzvkPCAaDiSxoQY5a0qchRjlWPEshwXHi+x8Gg9L5IL6M=
-  - secure: MDr12e/VPN/3rM/maEXXtLb9HEpYIz0qp3grx4upJnR0samcwuiOmO/8TkIRY4oxGwLl7sfE51b5P+2LiEhGInQqWl4EkTKtDjFLuGsakIFK6NUzLBN1K76tmVL0RGfIU+Abo4Y+LWyIBRn3mlp1KlIQlXZo0tHPACEfloHgDlw3YFpJM3kNWK5XlvsxetL2yudG7zTpJbAuJ37h4J38tIAs62x+b5bTaMOz7E0i+DdS8lm9XtIjiwQq6bMIBMFc/xkbfNf6u3MSJ/3Lbi5P0+rTCoVXC6byyzLBqrtYfu8+RLA7K2G/3/guZwQe7ERJgogIJN4rHhR83akUxvfNRutRN5/IwVVS9MKaVAzxQMn11io/Ju5EDBP0Kx7P6ZenJi816UCrew/1sC3cyElJ95zlCAga2pehZ9XumL0V5mwUuAKFu2DsncCh/q/Ci/gNK6OWoSVKekdjFFpeWPXLnriasRWeKmkGNPCHQIQeR747GPSqbS2o4TMh+0H4KzCJQV6p0vu8iH3D/up/QVzIAjb0KLrDXzZUZypk1z1KmHx9lcBRKMzIiyNLkKG35Shb6zhw13lACHlHYKoWwLpJ91Qn5kxr++BSC1JTlNdc8/s/Creqwpz50a4OwgroOZ+tGtnXOd1QCMfMQCHC+NlCnK8LYYkd54v02+LTdVFDMZo=
+  - secure: kbjHjTjelurgc2HJNiy5WbTMS4tTzKRGLF5sKVo2YQ27tK57fSbqm4dwgghkNX1DgVOg9P9m+IzGnRY+3f2Suwz8+rCZbi28pvfS0wLblyWgA526WBrY/W2ceh47IWmdIl3Y9xmzy9zia0hAux71JWY2kuhkxXxdSSmSj+p7bt9Mvptr/BxCdZvpmkt6vNHWw+7/dMqySMpk2hDIkb420haKOSuoWVcoXkBmF/Y2x7jg7EZgT2a6UutBl5c1F+G7RzibICBpdGn3FFKv05Kxwm2r9e4emOP0eYg65UQepvzG+TVSTe3IAY7SFEs5ax/3hYPObdXMTzlFk9Iak2vqNB7lDKgwAj83uWvn9nNERm+tKh652r6r++Xxf/CbpFmYeD0HQICC0/EkFTKgefhtb5GYzP9MBRiSUoj/oIsNAyMp8xbvhkBnYJj7ppXp0kFEnbiNjduseJ88yqtlNSFvaV0WtORUoB4Cqxj3wbaShFinXGGXjiXiCA7z8QNuJeqZ5oI1O3v/BNDDr5b/DOTsDwU9JBhMOCbBRp34uK+IugRKj1eXCMNz8I1IOFu8/iSeeC15Z1/77Yeb5a3MbdRqDf4Q8PY59L1EaRj0/dl6Mvl7EViSmzqfOUx80Y3Zx6oy2ALunVJ8kPIaFV+glo3WKJ3Hwcaj3ZN5IsJIrkfmUsY=
```

Comment:
What are these?

Reply:
new AZURE_ACCESS_TOKEN and AZURE_ACCOUNT_ID